SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/TouchSelection.swift

```diff
@@ -42,12 +42,13 @@ class TouchSelection {
 	func refillGrid() {
 		let lastIndex = min(phrases.count - 1, index + gridSize - 5)
 		//	functional grid cells are recognized by the four phrases mentioned below
-		var gridPhrases = Array(phrases[index...lastIndex]) + ["Oops","Next","Home","Done"]
-		if (gridPhrases.count < gridSize) {	//	add dummy elements
+		var gridPhrases = Array(phrases[index...lastIndex])
+		if (gridPhrases.count < gridSize - 4) {	//	add dummy elements
```

Comment:
It took me way too long to figure out why this was 4 haha.
Can you please just stick in a comment that says "4 is for the number of action buttons"

Reply:
oh I'm sorry....yes a comment is definitely needed there